### DISCUSSION TITLE
command or key to expand context?

### DISCUSSION THREAD

**Discussion by chuan2984** [pwntester/octo.nvim#523] [Q&A] [answered]
I just started usin this awesome plugin, is there a way to expand up/down like you can in github without doing gf(going to file)?

**Comment by williambdean** [comment]
Can you explain what you mean by expanding up and down

**Reply by chuan2984** [reply]
sure, you know how in github code review, theres that little plus sign you can click in a diff that will expand the window to include the previous X number of lines? you can do the same on the bottom with the arrow to see what comes after.

**Reply by williambdean** [reply, marked as answer]
They are just folds in the buffer. You should be able to use `zo` to open the fold

**Reply by chuan2984** [reply]
this is the answer, thanks!

**Reply by williambdean** [reply]
> this is the answer, thanks!

You might be able to unfold 10 lines only and set that to a keymap in octo buffer in order to act like UI
